fades/main.py

```diff
@@ -56,13 +57,48 @@
 """
 
 
-def _merge_deps(deps):
-    """Merge all the dependencies; latest dicts overwrite first ones."""
-    final = {}
-    for dep in deps:
+def consolidate_dependencies(needs_ipython, child_program,
+                             requirement_files, manual_dependencies):
+    """Parse files, get deps and merge them."""
+    # Deps specified first are overwritten by those specified last
```

Comment:
This comment is about the generic behaviour of the whole function. It should be part of the docstring.

Reply:
Yeah, I had some trouble trying to make it all fit in a single < 100 chars line :sweat_smile: Changed it.